perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -133,6 +144,30 @@ def Prepare(benchmark_spec):
   vms[0].RemoteCommand('sudo chmod 777 %s' % REMOTE_SCRIPT)
 
 
+def _SetupHostFirewall(benchmark_spec):
+  """Set up host firewall to allow incoming traffic.
+
+  Args:
+    benchmark_spec: The benchmark specification. Contains all data that is
+        required to run the benchmark.
+  """
+
+  client_vm = benchmark_spec.vms[0]
+  server_vm = benchmark_spec.vms[1]
+
+  if vm_util.ShouldRunOnExternalIpAddress():
```

Comment:
Running on external ips doesn't exclude internal ips - they can both be tested in the same run